sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -154,6 +156,7 @@ def _parse_conn_str(
         entity,
         str(shared_access_signature) if shared_access_signature else None,
         shared_access_signature_expiry,
+        bool(use_emulator=="true") if use_emulator else False
```

Comment:
the format of the localhost:port causes our conn_str parsing to break because previously we only supported this format via custom_endpoint